docs/integrations/destinations/mssql.md

```diff
@@ -115,6 +115,7 @@ Using this feature requires additional configuration, when creating the source.
 
 | Version | Date | Pull Request                                             | Subject                                                                                             |
 |:--------| :--- |:---------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| 0.1.20  | 2022-07-12 | [14649](https://github.com/airbytehq/airbyte/pull/14649) | Include lifecycle management                                            |
```

Comment:
State Lifecycle management?

Reply:
It refers to https://github.com/airbytehq/airbyte/pull/13670.